lib/push.js

```diff
@@ -44,7 +44,12 @@ module.exports = function (log, db) {
             if (device.pushCallback) {
               // send the push notification
               log.increment(LOG_UPDATE_INCREMENT_SEND)
-              request.post(device.pushCallback, function (err, response) {
+              request.post({
+                url: device.pushCallback,
+                headers: {
+                  'ttl': 0
```

Comment:
(A force-push may have eaten my previous comment about this)

Since this is a header, I think we should use a string value "0" here rather than a number, to avoid any potential for weirdness.